src/components/IconButton/IconButton.module.scss

```diff
@@ -0,0 +1,17 @@
+@use '../../styles/theme' as theme;
+
+.IconButton {
+  border-radius: 50%;
+  padding: 2px;
+  line-height: 0px;
+
+  cursor: pointer;
+  background: none;
+  border: none;
+  display: inline-block;
+  margin: 0px;
+
+  &:hover {
+    background-color: theme.$color-greyscale-transparency-3;
```

Comment:
Is there a benefit here to using SCSS `use` vs. using the CSS variable directly (`--color-greyscale-transparency-3)?

Reply:
The reason I went that way as that the build will fail if you use a value that doesn't exist. If you use the css var directly then you really won't know. It's still possible the theme will reference a css var that doesn't exist but it felt cleaner to have that part all collected in a single place. We might even be able to auto-generate the theme file?

Reply:
Cool - nice idea that seems very helpful! We'll look into theme engine next sprint